Sources/TuistKit/Services/DependenciesService.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistDependencies
+import TuistLoader
+import TuistSupport
+
+final class DependenciesService {
+    private let dependenciesController: DependenciesControlling
+    private let dependenciesModelLoader: DependenciesModelLoading
+
+    init(dependenciesController: DependenciesControlling = DependenciesController(),
+         dependenciesModelLoader: DependenciesModelLoading = DependenciesModelLoader())
+    {
+        self.dependenciesController = dependenciesController
+        self.dependenciesModelLoader = dependenciesModelLoader
+    }
+
+    func run(path: String?, method: InstallDependenciesMethod) throws {
+        switch method {
+        case .fetch: logger.info("Start fetching dependencies.", metadata: .section)
+        case .update: logger.info("Start updating dependencies.", metadata: .section)
+        }
+
+        let path = self.path(path)
+
+        let dependencies = try dependenciesModelLoader.loadDependencies(at: path)
+        try dependenciesController.install(at: path, method: method, dependencies: dependencies)
```

Comment:
Nitpick: If we are calling the actions "fetch" and "update", I think this method should be called "fetch" to be consistent. These are subtle things, but when there are many of them, they might compromise the readability of the code. Also, I'd recommend not doing anything if the user decides to update because update is not supported yet. If they try to update and we do fetch instead we are doing something that they don't expect us to do:

```swift
switch method {
  case .fetch:
    logger.info("We are starting to fetch the dependencies", metadata: .section)
    let dependencies = try dependenciesModelLoader.loadDependencies(at: path)
    try dependenciesController.fetch(at: path, method: method, dependencies: dependencies)
  case .update:
    logger.info("Updating dependencies is not yet supported.")
}
```

Reply:
I think that `update` is supported. `CarthageCommandGenerator` can build `carthage update` command.

Reply:
Right, the generator can generate the command, but can I update dependencies by running `tuist dependencies update`?

Reply:
I guess so. 

`tuist dependencies update` triggers `carthage update`. `carthage update` fetches the newest dependencies that matches requirements. 

`tuist dependencies fetch` triggers `carthage bootstrap`. carthage bootstrap` fetches dependencies that match `Cartfile.resolved`. If  `Cartfile.resolved` doesn't exist,  the newest dependencies will be fetched.

Reply:
I have just realized that I removed coping `Cartfile.resolved` from previous run (https://github.com/tuist/tuist/pull/2060#discussion_r539231951). I have to restore it to make sure that `tuist dependencies fetch` will fetch locked versions.

Reply:
@pepibumur after a few tests I have decided to mark it as `supported yet` like you suggested. 

I will revisit it the upcoming PRs.